pythoncalculator/multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
Hi Rob i think this should be x*y

Reply:
Have committed the change you suggested :)